README.md

```diff
@@ -368,6 +368,51 @@ end
 
 The code above was extracted from [the Neo4j Movies Demo](https://github.com/florinpatrascu/bolt_movies_elixir_phoenix), a Phoenix web application using this driver and the well known [Dataset - Movie Database](https://neo4j.com/developer/movie-database/).
 
+Note: as explained below, you don't need to convert your query result before hacing it  encoded in JSON. BoltSips provides Jason and Poison implementation to tackle this problem automatically.
+
+### About encoding
+Bolt.Sips povides solution for encoding your query result in different format.  
+For now, only JSON is supported.  
+
+There is two way of encoding data to json:
+- By using the helpers provided by the module `Bolt.Sips.ResponseEncoder`
+- Using your usual JSON encoding library. Bolt.Sips have implementation for both Jason and Poison. This means that query result can be automatically encoded by a library using Jason or Poison without further work (i.e Phoenix)  
+
```

Comment:
> Bolt.Sips have implementation for both Jason and Poison. This means that query result can be automatically encoded by a library using Jason or Poison without further work (i.e Phoenix) 

`Bolt.Sips` have implementation for: Jason and Poison. With this the query results can be automatically encoded by one of the libraries available; Jason or Poison. No further work is required when using a framework like: Phoenix, for example.

Reply:
Updated.
I've to stop thinking in french better writing in english!